revapi-maven-plugin/src/site/site.xml

```diff
@@ -4,21 +4,6 @@
     <name>${project.name}</name>
     <href>${project.url}</href>
   </bannerLeft>
-  <body>
-    <menu name="Project Details">
-      <item name="Goals" href="plugin-info.html"/>
```

Comment:
This disappeared from your changes (it's a page generated by maven from the code). Would there be a way to at least have it included in the bottom section of the page with the rest of the links somehow?